willow/lib/tasks/reindex_test_data.rake

```diff
@@ -0,0 +1,88 @@
+namespace :willow do
+  desc "Reindex Willow test data (do not run on production!)"
```

Comment:
Actually; not running this on production is mainly about performance rather than data corruption. It takes a long time to re-index every record, and if you have thousands of them you could be waiting a long time.

...however, it may be useful to run this method on a _production-like_ environment, such as running in production mode with a handful of objects. I have updated the comment to make this clearer.